sos/cleaner/__init__.py

```diff
@@ -132,13 +133,14 @@ def validate_map_file(self):
         else:
             with open(self.opts.map_file, 'r') as mf:
                 try:
-                    json.load(mf)
+                    _conf = json.load(mf)
                 except json.JSONDecodeError:
                     self.log_error("ERROR: Unable to parse map file, json is "
                                    "malformed. Will not load any mappings.")
                 except Exception as err:
                     self.log_error("ERROR: Could not load '%s': %s"
                                    % (self.opts.map_file, err))
+        return _conf
```

Comment:
As we newly load the map file here, I suggest renaming the `validate_map_file` to e.g. `load_map_file`.

Reply:
Ack, good idea.